tests/test_helper.js

```diff
@@ -0,0 +1,366 @@
+const Recipe = require("../models/Recipe");
```

Comment:
This file is incredibly hard to read. I would break up into multiple files. Also, the data for tests doesn't need to be so detailed.

Reply:
Also, it might be better to use a library? I found mockingoose after a quick search